gen/enum_test.go

```diff
@@ -335,6 +335,28 @@ func TestUnmarshalTextReturnsErrorOnInvalidValue(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestMarshalTextReturnsValue(t *testing.T) {
+	tests := []struct {
+		title string
+		e     te.EnumDefault
+		text  string
```

Comment:
nit: for field naming for table tests, we've preferred to name values `give`
or `giveFoo`, `giveBar` for the input and `want` or `wantFoo`, `wantBar` for
the output. It clearly separates what the input of the test case is from the
expected output, especially when there are multiple.

Reply:
Ah that's a nice convention!